packages/common/editors/EditorContainer.js

```diff
@@ -336,15 +313,17 @@ class EditorContainer extends React.Component {
     const style = { position: 'absolute', height, width, left, top, zIndex: zIndexes.EDITOR_CONTAINER };
     return (
       <EditorPortal>
-        <div style={style}
-          ref={this.setContainerRef}
-          className={this.getContainerClass()}
-          onKeyDown={this.onKeyDown}
-          onContextMenu={this.handleRightClick}
-        >
-          {this.createEditor()}
-          {this.renderStatusIcon()}
-        </div>
+        <ClickOutside
+          onClickOutside={this.commit}>
```

Comment:
Could keep this on the same line as the opening tag.

Reply:
Done